relay-server/src/actors/upstream.rs

```diff
@@ -181,18 +245,22 @@ fn handle_response(
     Box::new(future)
 }
 
-pub struct UpstreamRelay {
-    backoff: RetryBackoff,
-    config: Arc<Config>,
-    auth_state: AuthState,
-}
-
 impl UpstreamRelay {
     pub fn new(config: Arc<Config>) -> Self {
+        //TODO discus size (probably shouldn't be very big, if notifications accumulate we
+        // have a problem, unbounded probably a bad idea)
+        let (sender, receiver) = mpsc::channel(1000);
         UpstreamRelay {
+            pump_http_queue_notifications: Some(receiver),
+            http_request_finished_notifier: sender,
             backoff: RetryBackoff::new(config.http_max_retry_interval()),
             config,
             auth_state: AuthState::Unknown,
+            // TODO get the real value from a config and use it with ClientConnector::limit
+            max_inflight_requests: 100,
```

Comment:
Please add a config value to `config.limits`.

Reply:
Yes, after looking a bit more I realised that we already have a configuration variable for this `Limits::max_concurent_requests` which is already used to configure the `ClientConnector::limit()`.

Changed from hardcoded value to `Config::max_concurent_requests()`

Reply:
That's perfect, of course that's the one we want anyway.